### ISSUE TITLE
When loading/creating world it sits at 0%

### ISSUE THREAD

**Issue by minimurgle** [Darkosto/SevTech-Ages#1044]
## Issue / Bug
When loading/creating world it sits at 0%

## Current Behavior
Downloaded the modpack and tried to create a world. It just sat at 0% completion and was using 0.3% CPU. For reference the title screen uses 2-3%
I let it run on that screen for about half an hour. It didn't crash and was still responsive. Hitting Esc brought me back to the main menu just fine.
**Edit** *(March 29 at 10:11am):* I've notived that when I close the game it leaves the Java Platform SE Binary open. It also doesnt clear any of the cached RAM so even after I close it the pack uses around 6 gb of ram. This happens both upon closing it through the menu and with alt f4.

## Steps to Reproduce (for bugs)
1. Launch game
2. Create new world

## Attempted Fixes
* Twitch Launcher repair minecraft
* Reinstall minecraft manuall
* Reinstall modpack
* Run as admin
* Restart PC
* Search for issue here and on google 

## Your Environment
* Modpack Version: 3.0.3
* Java Version: Java 8 Update 161 (64-bit)
* Launcher Used: Twitch. In the twitch settings it's set to use the jar launcher
* Resourcepack Enabled?: No
* Optifine Installed?: No
* Total PC Ram: 20GB
* Ram Allocated To Pack: 10GB

**Comment by Trinsdar** [comment]
To me it sounds like you have 10 gigs allocated to the pack. Java screws up when allocated too much ram, so try allocating only six gigs of ram.

**Comment by 630R63** [comment]
I have 10 gigs of ram (out of 32) allocated to the pack. No issues on my side.

**Comment by minimurgle** [comment]
@Trinsdar Edited the wording on that. Sorry didn't realize how weird it was at the time. 
I just tried running it with a few different amounts of RAM. 6,7,8,10,and 12GB. Same results every time. Thanks though.

**Comment by minimurgle** [comment]
Updated original post with some more info under current behavior.

**Comment by leosia** [comment]
I have the same issue, but I run my world after first installation of pack, only once, and then I cannot run/create any world.
- It's happend only in this pack, I've tried others, the same size packs and all works fine. 
- I reinstall my Windows 7. Again, I create a world and play for a while, but next time world not loaded/created.
- I change to Windows 10, not helped
- I change launcher to ATLauncher, not helped

**Comment by Viba** [comment]
Had the same problem, disabling ImprovedBackpacks resolved it for me.

**Comment by leosia** [comment]
@Viba, thanks for the tip, disabling this mod helps with running world, but it cause that some of recipes are messed up, like wood planks I can now make in vanilla way. 
I tryied Java 1.8.025, worlds not running/creating.

**Comment by Viba** [comment]
I struggled with this today again, for some reason reinstalling SevTech via Twitch launcher works. But if I restart the game I might have to reinstall it again.

**Comment by leosia** [comment]
A question, did anyone with this issue changed key bindings? Or maybe more precise, changed to some keys to additional mouse buttons? 
I want to check this solution #1180, because I do the same thing, I change "sneak" button to my mouse additional button in all instances. 
I checked this and I cannot load my old world but I can create new one. I tryied to load the world and quit for few times, all works fine, maybe it will help.

**Comment by Viba** [comment]
I did that among some of the fix attempts, for me it didn't help. I always change sprint and sneak to mouse thumb buttons.

**Comment by leosia** [comment]
I start new world, played for some time, everything is fine, besides of course assigninga additional mouse buttons. I don't want to try it because I want to play this awsome pack. Only wish to @Darkosto try to fix this, beacuse it's so difficult to me to play without additional mouse buttons. Thanks for help.

**Comment by Viba** [comment]
It works if you launch the game without the thumb buttons assigned, once it is running you an assign them. Just remember to reset them before you save and quit the game, otherwise it gets stuck again.

**Comment by minimurgle** [comment]
Didnt seem to be an issue with Java or the thumb buttons. Changed the java version and the thumb buttons weren't assigned. 

Though it does seem to be working now with the 3.0.4

**Comment by justinrusso** [comment]
Since this seems to be resolved, I'm going to be closing this issue. Please submit a new issue if anything else comes up.
